src/game/boe.specials.cpp

```diff
@@ -1855,7 +1855,7 @@ void special_increase_age(long length, bool queue) {
 			if(univ.town->timers[i].time > 0) {
 				short time = univ.town->timers[i].time;
 				bool need_redraw = false;
-				for(unsigned long j = age_before + (time == 1); j <= current_age; j++)
+				for(unsigned long j = age_before + 1; j <= current_age; j++)
```

Comment:
So, if I understand correctly, the bug is that a timer could be called on the turn after it was called? And this definitely fixes it? Just double-checking since the commit message contains "try".

As an aside: the original source code looks dramatically different so it's hard to compare, but setting the timer's time to 0 should also prevent it from double-triggering.

Reply:
Yes, I tried to avoid double triggering (which could appear for this reason or another) ; the previous code was not logical, ie. when length=1 and time!=1, the execution of the code was:
```
for(unsigned long j = party.age-1 ; j <= party.age; j++) ...
party.age++
for(unsigned long j = party.age-1 ; j <= party.age; j++) ...
```
( ie. we tried to trigger each timer twice )